microsoft/electionguard-ui
# 🎞️ Manifest Preview

### Issue

Fixes #122 

### Description
Parses manifest uploaded data and displays it on the preview page

### Testing
1. Log in 
2. Setup Election
3. Navigate through steps until Upload Election Manifest
4. Select the new manifest.json file in /data

Expected: app navigates to the preview page which should contain:

Name: Hamiltion County General Election
Number of Contests: 8
Number of Ballot Styles: 11
Start Date: 3/1/2020 8:00:00 AM
End Date: 3/1/2020 8:00:00 PM

Like this:

![image](https://user-images.githubusercontent.com/1769905/151861669-6d568592-419c-4030-b387-74855890298e.png)
